### PR TITLE
Explain Stream Sequence Number and TSN  in more depth

### PR DESCRIPTION
**Pull request by wawesomeNOGUI**
Hi! I wanted to add some clarity to the Stream Sequence Number and TSN. Let me know if too wordy, or if anything should be added.

### CODE REVIEW

File: content/docs/07-data-communication.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ anything over the data channel, this is how it is exchanged.
 Stream Sequence Number.
 
 `B` and `E` are the beginning and end bits. If you want to send a
-message that is too large for one DATA chunk it needs to be fragmented.
+message that is too large for one DATA chunk it needs to be fragmented into multiple DATA chunks that are sent in seperate packets.
```
Comment (mogren) [inline content/docs/07-data-communication.md:182]: Nit: seperate -> separate 

Also, how about just "into multiple DATA chunks sent in separate packets."?

**Review by mogren — CHANGES_REQUESTED** [review]
Awesome, thanks for improving this! Just a minor nit.

File: content/docs/07-data-communication.md (hunk 1 above)
Comment (wawesomeNOGUI) [reply content/docs/07-data-communication.md:182]: Hi! Thanks for the response, that sounds good, I’ll edit that in

**Review by mogren — APPROVED** [review]
Great, thanks!
